### DISCUSSION TITLE
Connect from lan

### DISCUSSION THREAD

**Discussion by daschmidt1994** [nextcloud/all-in-one#4785] [Questions] [answered]
Hello,

I have installed nextcloud aio with the 11000 proxy. Now I can connect from my.domain.at to the nextcloud. Have the proxy set with cloudflare tunnel. This means that nextcloud runs locally at my home. Cloudflare is only used for the connection from outside. However, when I am in the local network I would like to log in directly to my nextcloud instance so that I don't have to send all the files over the internet first. It runs in an lxc pihole and nginxproxymanager. In pihole I specified that my.domain.at should redirect to the nginx IP. In nginx proxy manager I set http ip port, but unfortunately I can't access my nextcloud with the IP from my home network. What is the configuration error here or can it not work like this?

**Comment by szaimen** [comment]
Hi, please read https://github.com/nextcloud/all-in-one?tab=readme-ov-file#how-can-i-access-nextcloud-locally

**Reply by daschmidt1994** [reply]
yes this I have already read.

Work it if I link my cloudfare domain cloud.example.com to 192.168.178.220(this is my nginx proxy manager) and from there then to my nc aio 192.168.178.90:11000?

**Reply by szaimen** [reply, marked as answer]
This only works if the certificate handling is done locally and not via cloudflare tunnel.
